algorithms/afl: Disable memory limit by default (#57)
fuzzuf/fuzzuf: docs/algorithms/afl/algorithm_en.md

    - `--exec_timelimit_ms=1234`
        - Specifies a time limit per PUT execution in milliseconds. The default time limit is 1 second (i.e. 1000 ms).
    - `--exec_memlimit=1234`

>         - Specifies the amount of memory available per PUT execution in megabytes. The memory limit is disabled by default.

Reviewer: ```suggestion
        - Specifies the amount of memory available per PUT execution in megabytes. No memory limits are enforced by default.
```
